catalog/admin/module/catalog-admin-module-sources/src/main/webapp/js/model/Service.js

```diff
@@ -326,16 +329,16 @@ define(function (require) {
 
     Service.Response = Backbone.AssociatedModel.extend({
         url: "/jolokia/exec/org.codice.ddf.catalog.admin.poller.AdminPollerServiceBean:service=admin-source-poller-service/allSourceInfo",
-        relations: [
-            {
-                type: Backbone.Many,
-                key: 'value',
-                relatedModel: Service.Model,
-                includeInJSON: false
-            }
-        ]
+        relations: [{
+            type: Backbone.Many,
+            key: 'value',
+            relatedModel: Service.Model,
+            includeInJSON: false
+        }],
+        parse: function (data) {
+            return data;
```

Comment:
Why did you need to add this function in here? This is the backbone default action anyways.